src/main/kotlin/no/nav/pdfgen/Helpers.kt

```diff
@@ -102,9 +103,22 @@ fun registerNavHelpers(handlebars: Handlebars) {
             val checkfor = options.param(0, null as String?)
 
             val contains = list
-                ?.map { Context.newContext(options.context, it) }
-                ?.any { ctx -> !options.isFalsy(ctx.get(checkfor)) }
-                ?: false
+                    ?.map { Context.newContext(options.context, it) }
+                    ?.any { ctx -> !options.isFalsy(ctx.get(checkfor)) }
+                    ?: false
+
+            if (contains) {
+                options.fn()
+            } else {
+                options.inverse()
+            }
+        })
+
+        registerHelper("contains_all", Helper<ArrayNode> { list, options ->
```

Comment:
Er det mulig å unngå bruk av ArrayNode her? Kan man bruke en listetype?

Reply:
Ja jeg syns også det var stygt, men hvis jeg bruke `Iterable<String>`  har hvert element dobbelescapet strenger :( For å få "vanlige" strenger så måtte jeg kalle `textValue()` på JsonNode noden, eller fjerne escapene selv. Litt usikker på hvorfor dette ikke er problem for de andre helperne, men er åpen for tips

Reply:
Du kan prøve med Iterable<Any>, mulig den bare kjører JsonNode#toString på alle elementer når man ber om string. Vet ikke helt om man bare ender opp med Iterable<JsonNode> i stedet da

Reply:
toString tar med anførselstegn. 
Ser jeg var upresis i forrige kommentar. Med dobbelescapet strenger mente jeg at anførselstegnene ble med i strengen. 
Kan prøve med Any uten å gjøre til string.

Reply:
Mente at jeg trodde JsonNode#toString var kilden til escaped strings

Reply:
ser ut som equals metoden til `TextNode` returnerer false hvis den sammenlignes med noe annet en TextNode: 
```
public boolean equals(Object o)
    {
        if (o == this) return true;
        if (o == null) return false;
        if (o instanceof TextNode) {
            return ((TextNode) o)._value.equals(_value);
        }
        return false;
    }
``` 

derfor returnerer `containsAll()` false når det sammenlignes med en String :(